activiti-cloud-query-service/activiti-cloud-starter-query/src/main/resources/config/query/liquibase/changelog/06-alter.pg.schema.7.1.0-m9.sql

```diff
@@ -0,0 +1,4 @@
+alter table process_instance
```

Comment:
Same here, the suffix of the file name should be `m10`.

Reply:
Thanks, renamed both sql file names